kops/carbonplan.jsonnet

```diff
@@ -0,0 +1,73 @@
+local cluster = import "./libsonnet/cluster.jsonnet";
+local ig = import "./libsonnet/instancegroup.jsonnet";
+
+local zone = "us-west-2a";
+local nodes = [
+    {spec+: { machineType: "r5.large" }}, // 2CPU, 16G RAM
+    {spec+: { machineType: "r5.xlarge" }}, // 4CPU, 32G RAM
+    {spec+: { machineType: "r5.2xlarge" }}, // 8CPU, 64G RAM
+    {spec+: { machineType: "r5.8xlarge" }} // 32CPU, 256 RAM
+];
+
+local data = {
+    cluster: cluster {
+        metadata+: {
+            name: "carbonplanhub.k8s.local"
+        },
+        spec+: {
+            configBase: "s3://2i2c-carbonplan-kops-state"
+        },
+        _config+:: {
+            zone: zone,
+            masterInstanceGroupName: data.master.metadata.name
+        }
+    },
+    master: ig {
+        metadata+: {
+            labels+: {
+                "kops.k8s.io/cluster": data.cluster.metadata.name
+            },
+            name: "master"
+        },
+        spec+: {
+            machineType: "t3.medium",
+            subnets: [zone],
+            nodeLabels+: {
+                "hub.jupyter.org/pool-name": "core-pool"
```

Comment:
Je, this one was the label I was missing in my kops cluster (copied from the pangeo hubs).